lib/manage-prototype-handlers.js

```diff
@@ -545,14 +549,15 @@ async function getPluginsModeHandler (req, res) {
   })
 }
 
-const prototypeTimestamp = Date.now()
+function setKitRestarted () {
```

Comment:
When you're using this function you pass in a value which then gets ignored `setKitRestarted(true)`.

Reply:
This now uses this value as it should.